announcing-community-cdn.rst

```diff
@@ -0,0 +1,80 @@
+Shipping a CDN on Read the Docs Community
```

Comment:
Meta tags are missing here (`.. post` with author's name and date, and `.. meta` with SEO info)